.github/workflows/examples.yml

```diff
@@ -0,0 +1,95 @@
+name: Examples
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  # NOTE: github.event.head_commit.message is not available on pull_request events
+  head-commit:
+    name: Inspect head commit
+    runs-on: ubuntu-latest
+    outputs:
+      message: ${{ steps.commit.outputs.message }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{github.event.pull_request.head.sha}}
+      - name: Inspect
+        id: commit
+        run: echo "message=$(git show -s --format=%s)" >> "$GITHUB_OUTPUT"
+
+  convert:
+    needs: [head-commit]
+    if: "${{needs.head-commit.outputs.message != 'docs(examples): converted from e2es'}}"
+    strategy:
+      matrix:
+        e2e:
+          - extra-fields
```

Comment:
This will create workflows for each example which might cause a bigger queue. But instead maybe we can run this once and configure converted e2es in a file, no?